src/XUnity.AutoTranslator.Plugin.BepInEx-IL2CPP/AutoTranslatorPlugin.cs

```diff
@@ -31,7 +31,23 @@ public AutoTranslatorPlugin()
 
          _configPath = Path.Combine( ConfigPath, "AutoTranslatorConfig.ini" );
 
-         Il2CppProxyAssemblies.Location = Path.Combine( Paths.BepInExRootPath, "interop" ); // Il2CppInteropManager.IL2CPPInteropAssemblyPath is internal...
+         Il2CppProxyAssemblies.Location = GetIL2CPPInteropAssemblyPath();
+      }
+
+      private string GetIL2CPPInteropAssemblyPath()
+      {
+         // Il2CppInteropManager.IL2CPPInteropAssemblyPath is internal...
+         try
+         {
+            var Il2CppInteropManagerType = Type.GetType( "BepInEx.Unity.IL2CPP.Il2CppInteropManager, BepInEx.Unity.IL2CPP" );
+            Log.LogInfo( Il2CppInteropManagerType );
+            return (string)Il2CppInteropManagerType.GetProperty( "IL2CPPInteropAssemblyPath", BindingFlags.Static | BindingFlags.NonPublic ).GetValue( null );
```

Comment:
```suggestion
            return (string)Il2CppInteropManagerType.GetProperty( "IL2CPPInteropAssemblyPath", BindingFlags.Static | BindingFlags.NonPublic | BindingFlags.Public ).GetValue( null );
```
The property might be changed to public in the future.